benefits/core/migrations/0002_sample_data.py

```diff
@@ -108,12 +108,12 @@ def load_sample_data(app, *args, **kwargs):
         selection_label=_("eligibility.pages.index.sample.label"),
         selection_label_description=None,
         start_headline=_("eligibility.pages.start.sample.title"),
```

Comment:
Shouldn't this key be `eligibility.pages.start.sample.headline` and the following be `eligibility.pages.start.sample.sub_headline`?

Reply:
Yes, you're right. Thank you